packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagegroup/_index.scss

```diff
@@ -0,0 +1,22 @@
+.dx-chat-message-group {
+  display: grid;
+  align-items: start;
+}
+
+.dx-chat-message-group-information {
+  display: flex;
+  grid-row: 1;
+}
+
+.dx-chat-message-group-alignment-start {
+  justify-items: start;
+  grid-column: 2;
```

Comment:
Seems, this property doesn't work

Reply:
[b5bbab3e3d2663f50b15f1c50726ddb07f6b6094]